### DISCUSSION TITLE
Creating new accounts with + in email is not allowed

### DISCUSSION THREAD

**Discussion by Kamelot73** [civitai/civitai#1370] [General]
I can´t sign in to my Civitai account. There is no + in my e-mail adress. 
Creating new accounts with + in email is not allowed

**Comment by hardreddata** [comment]
I struck this recently.

I had two similar but unintentional logins like user.name@mail.com and then later username@mail.com. The first one was working but the second one gave an error.

Deleting the first account let me into the second one.
